contracts/v1/Identity.sol

```diff
@@ -110,6 +114,31 @@ contract Identity is Named, Versioned, ContractStatus, Initializable {
         if (purpose == OPERATIONAL_KEY) ids.removeOperationalKeyIdentityId(key);
     }
 
+    function addOperationalWallets(uint72 identityId, address[] calldata operationalWallets) external onlyContracts {
+        IdentityStorage ids = identityStorage;
+
+        bytes32 operationalKey;
+        bytes32 attachedKey;
+
+        for (uint i; i < operationalWallets.length; ) {
```

Comment:
Check for length ? @u-hubar

Reply:
We have it in Profile, where we use this function